servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/RuleDescriptionSerializer.java

```diff
@@ -349,6 +351,8 @@ private static Filter parseCorrelationFilterFromElement(Element filterElement) {
                     case "ContentType":
                         filter.setContentType(element.getFirstChild().getNodeValue());
                         break;
+                    default:
```

Comment:
[SpotBugs-P2]
This method contains a switch statement where default case is missing. Usually you need to provide a default case.